gettsim/config.py

```diff
@@ -104,6 +105,15 @@
     "grundr_bew_zeiten": int,
     "priv_rente_m": float,
     "schwerbeh_g": bool,
+    "rententrechtl_zeit": float,
```

Comment:
These are quite some new variables. I think we should think another time whether we need all of them (e.g. can we use `grundr_zeiten`?)

Reply:
we cannot use grundr_zeiten they are different from the other "wartezeiten". actually I think we should decompose them so it is more transparent to the user what is required here.

Reply:
That sounds good!